<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTabTester</class>
 <widget class="QWidget" name="FormTabTester">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Tab Properties</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="isEnabled">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Enabled</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="isVisible">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>&amp;Visible</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>&amp;Create Tab</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnDelete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Delete Tab</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBarrage">
       <property name="text">
        <string>&amp;Barrage</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>isEnabled</tabstop>
  <tabstop>isVisible</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
